inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/source/kafka/KafkaSource.java

```diff
@@ -67,7 +68,7 @@ public class KafkaSource extends StreamSource {
     private String timestampMillis;
 
     @ApiModelProperty(value = "The strategy of auto offset reset", notes = "including earliest, specific, latest (the default), none")
-    private String autoOffsetReset;
+    private KafkaOffset autoOffsetReset = KafkaOffset.LATEST;
```

Comment:
`AutoOffsetReset` also exists in `KafkaSourceRequest` and `KafkaSourceDTO`. If kafka is used as the source during data synchronization, it will become unavailable.

Reply:
Just set the default value of source to private `String autoOffsetReset=KafkaOffset. LATEST. getName()`;

Reply:
OK